docs/examples/Sensor_Platform_Simulation.py

```diff
@@ -0,0 +1,381 @@
+
+# Some general imports and set up
+from datetime import datetime
+from datetime import timedelta
+
+import numpy as np
+
+# Stone Soup imports:
+from stonesoup.types.state import State, GaussianState
+from stonesoup.types.array import StateVector, CovarianceMatrix
+from stonesoup.models.transition.linear import (
+    CombinedLinearGaussianTransitionModel, ConstantVelocity)
+from stonesoup.models.measurement.nonlinear import CartesianToElevationBearingRange
+from stonesoup.updater.kalman import UnscentedKalmanUpdater
+from stonesoup.predictor.kalman import UnscentedKalmanPredictor
+from stonesoup.deleter.time import UpdateTimeStepsDeleter
+from stonesoup.tracker.simple import MultiTargetTracker
+from matplotlib import pyplot as plt
+
+# Define the simulation start time
+start_time = datetime.now()
+
+# %%
+# Create a Platform
+# -----------------
+# The first element we need to create is a platform. For this first example we will build a static (or *fixed*) platform
+# which is located at the origin. For this example we are going to work in a 6-dimensional state space our platform will
+# have the following :math:`\vec{x}`.
+#
+# .. math::
+#           \begin{align}\vec{x} = \begin{bmatrix}
+#                                   x\\ \dot{x}\\ y\\ \dot{y}\\ z\\ \dot{z} \end{bmatrix}
+#                                = \begin{bmatrix}
+#                                   0\\ 0\\ 0\\ 0\\ 0\\ 0 \end{bmatrix}\end{align}
+#
+# Because the platform is static we only need to define :math:`(x, y, z)`, any internal interaction with the platform
+# which requires knowledge of platform velocity :math:`(\dot{x}, \dot{y}, \dot{z})` will be returned :math:`(0, 0, 0)`.
+
+# First import the fixed platform
+from stonesoup.platform.base import FixedPlatform
+
+# Define the initial platform position, in this case the origin
+platform_state_vector = StateVector([[ 0], [0], [0]])
+position_mapping = (0, 1, 2)
+
+# Create the initial state (position, time), notice it is set to the simulation start time defined earlier
+platform_state = State(platform_state_vector, start_time)
+
+# create our fixed platform
+platform = FixedPlatform(states=platform_state,
+                         position_mapping=position_mapping)
+
+# %%
+# We have now created a platform within Stone Soup and located it at the origin of our state space. As previously stated
+# the platform will have a velocity :math:`(\dot{x}, \dot{y}, \dot{z})` of :math:`(0, 0, 0)` which we can check:
+platform.velocity
+
+# %%
+# We can also query the platform orientation:
+platform.orientation
+
+# %%
+# Create a Sensor
+# ---------------
+# Now that we have a platform the next step is to create a sensor which can be added to the platform. In this example a
+# Radar will be created which is capable of measuring the range, bearing and elevation of the target relative to the
+# sensor.
+#
+# The :class:`~.RadarElevationBearingRange`  provides a sensor wrapper around the
```

Comment:
Wrong class name here.

Reply:
Hangover from separation of this from proposed naming convention changes to radar, good spot